master/security/protect-hosts.md

```diff
@@ -0,0 +1,185 @@
+- **GlobalNetworkPolicy**
+- **FelixConfiguration** resource with parameters:
+  - **FailsafeInboundHostPorts** 
+  - **DefaultEndpointToHostAction**
+
+### Concepts
+
+#### Hosts and workloads
+
+In the context of Calico configuration, a **workload** is a virtualized compute instance, like a VM or container. A **host** is the computer that runs the hypervisor (for VMs), or container runtime (for containers). We say it “hosts” the workloads as guests.
+
+#### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can use Calico network policy to secure these interfaces (called host endpoints). Calico host endpoints can have labels, and they work the same as labels on workload endpoints. The network policy rules can apply to both workload and host endpoints using label selectors. 
+
+#### Failsafe rules
+
+It is easy to inadvertently cut all host connectivity because of non-existent or misconfigured network policy. To avoid this, Calico provides failsafe rules with default/configurable ports that are open on all host endpoints.
+
+#### Default behavior of workload to host traffic
+
+By default, Calico blocks all connections from a workload to its local host. You can control whether connections from a workload endpoint to its local host are dropped, returned, or accepted using a simple parameter.
+
+Calico allows all connections from processes running on the host to guest workloads on the host. This allows host processes to run health checks and debug guest workloads.
+
+#### Default behavior of external traffic to/from host
+
+If a host endpoint is added and network policy is not in place, the Calico default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules). For host endpoints, Calico blocks traffic only to/from interfaces that it’s been explicitly told about in network policy. Traffic to/from other interfaces is ignored.
+
+#### Other host protection
+
+In terms of design consistency in Calico, you may wonder about the following use cases.
+
+**Does Calico protect a local host from workloads?**<br> 
+Yes. DefaultEndpointToHostAction controls whether or not workloads can acesss their local host.<br>
+
+**Does Calico protect a workload from the host it is running on?**<br> 
+No. Calico allows connections the host makes to the workloads running on that host. Some orchestrators like Kubernetes depend on this connectivity for health checking the workload. Moreover, processes running on the local host are often privileged enough to override local Calico policy. Be very cautious with the processes that you allow to run in the host's root network namespace.</br>
+
+### Before you begin...
+
+If you are already running Calico for Kubernetes, you are good to go. If you want to install Calico on a bare-metal machine for host protection only see, [Bare metal hosts]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/installation/binary-mgr).
```

Comment:
> machine for host protection only see,

Will fix in a subsequent PR. This should be "machine for host protection only, see" (comma moved)